### PR TITLE
[docs] Update ROS info

### PR DESCRIPTION
**Pull request by rajat2004**

### CODE REVIEW

File: docs/airsim_ros_pkgs.md [hunk 1]
```diff
@@ -25,7 +25,13 @@ cd AirSim;
 
 ```
 cd ros;
-catkin_make; # or catkin build
+catkin build; # or catkin_make
```
Comment (rajat2004) [inline docs/airsim_ros_pkgs.md:28]: `catkin build` is the recommended way over `catkin_make`

File: docs/airsim_ros_pkgs.md [hunk 2]
```diff
@@ -25,7 +25,13 @@ cd AirSim;
...
+```
+
+If your default GCC isn't 8 or greater (check using `gcc --version`), then compilation will fail. In that case, use `gcc-8` exlicitly as follows-
```
Comment (madratman) [inline docs/airsim_ros_pkgs.md:31]: explicitly

File: docs/airsim_ros_pkgs.md (hunk 1 above)
Comment (madratman) [reply docs/airsim_ros_pkgs.md:28]: https://catkin-tools.readthedocs.io/en/latest/installing.html please add a note linking to installation steps for catkin tools

File: docs/airsim_ros_pkgs.md (hunk 2 above)
Comment (rajat2004) [reply docs/airsim_ros_pkgs.md]: fixed

File: docs/airsim_ros_pkgs.md (hunk 1 above)
Comment (rajat2004) [reply docs/airsim_ros_pkgs.md:32]: Added in Setup

Comment (madratman) [reply docs/airsim_ros_pkgs.md:32]: cool, thanks, merging

**Comment by rajat2004** [comment]
Thanks!

**Pull request merged**